service/src/test/kotlin/no/nav/su/se/bakover/service/vedtak/VedtakServiceImplTest.kt

```diff
@@ -161,6 +169,59 @@ internal class VedtakServiceImplTest {
         ).kopierGjeldendeVedtaksdata(UUID.randomUUID(), LocalDate.EPOCH.plusDays(7)) shouldBeLeft KunneIkkeKopiereGjeldendeVedtaksdata.UgyldigPeriode(Periode.UgyldigPeriode.FraOgMedDatoMåVæreFørsteDagIMåneden)
     }
 
+    @Test
+    fun `henter tidligere informasjon for overlappende vedtak`() {
+        val vedtak1 = vedtakSøknadsbehandlingIverksattInnvilget(Stønadsperiode.create(Periode.create(1.januar(2021), 31.desember(2021)))).copy(
+            opprettet = Tidspunkt.now(fixedClock)
+        )
+        val vedtak2 = vedtakSøknadsbehandlingIverksattInnvilget(Stønadsperiode.create(Periode.create(1.januar(2021), 31.desember(2021)))).copy(
```

Comment:
Det gir ikke mening at vi har overlappende søknadsbehandlingsvedtak. Kan vi bytte de 2 siste til revurderingsvedtak?

Reply:
Hehe har ikke noe å si for testen, men enig i att det er rart som koncept. Bytter! 😊